src/EmailAddress.ts

```diff
@@ -320,11 +320,11 @@ export class EmailAddress implements IEquatable, IJsonStringifyable {
      * @param {string} s A string containing email address to convert.
      * @returns {EmailAddress} email address if valid, otherwise throws.
      */
-    public static parse(s: string | null | undefined): EmailAddress | null {
+    public static parse(s: string | null | undefined): EmailAddress | undefined {
```

Comment:
Waarom returnt deze functie ook undefined? Want als de parse goed gaat komt er een EmailAddress uit en anders een error, wanneer zou er undefined uit komen?

Reply:
Let's stick to English here. ;)

It returns undefined if `null`, `undefined`, `''`, are whitespace is provided. Those are considered as valid import reflecting a not set email address.